### ISSUE TITLE
Weather applet feature request

### ISSUE THREAD

**Issue by Yasen6275** [mate-desktop/mate-applets#178]
Is it posible to copy/fork xfce4 weather applet meteo sations sources. Seems to be a bit reacher than mates one.

**Comment by zaps166** [comment]
Good idea, but currently for xfce4-weather-plugin and xfce4-whiskermenu-plugin you can use: https://github.com/zaps166/mate-xfce4-panel-plugin-loader-applet

**Comment by raveit65** [comment]
Well, if we switch to gtk3 in future, maybe 1.16 release, than we will use libgnomeweather.
For the moment it sounds like a good idea to update weather sources.

**Comment by LinuxJoeUser** [comment]
Please, whatever means is most suitable, I would love to see the Mate weather app as good and complete as the XFCE one, `xfce4-weather-plugin`.
